src/stars.css

```
@@ -0,0 +1,88 @@
+.stars {
+  display: flex;
+  width: 96px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(n) {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 :first-child {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star-active.svg);
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 :not(:first-child) {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star-active.svg);
+  height: 16px;
+  width: 16px;
+}
```

Continued from previous comment, do this for the rest of stars--N classes
```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2) {
  background-image: url(./images/star-active.svg);
}
```